recipe/build.sh

```diff
@@ -149,7 +149,7 @@ if [[ ${HOST} =~ .*linux.* ]]; then
                 -qt-pcre \
                 -qt-xcb \
                 -qt-xkbcommon \
-                -xkb-config-root $PREFIX/lib \
+                -xkb-config-root /usr/share/X11/xkb \
```

Comment:
@mingwandroid I would like to get your opinion here.

Reply:
Seems OK? I think we had that.. I remember trying to use a pathlist with a : between but it doesn't work.

Reply:
Build locally with this not. Hold on...